QuantConnect.TradeStationBrokerage/TradeStationBrokerage.cs

```diff
@@ -357,13 +369,13 @@ public override bool PlaceOrder(Order order)
     /// <returns>
     /// A <see cref="CrossZeroOrderResponse"/> object indicating the result of the order placement.
     /// </returns>
-    public override CrossZeroOrderResponse PlaceCrossZeroOrder(CrossZeroOrderRequest crossZeroOrderRequest, bool isPlaceOrderWithLeanEvent)
+    protected override CrossZeroOrderResponse PlaceCrossZeroOrder(CrossZeroOrderRequest crossZeroOrderRequest, bool isPlaceOrderWithLeanEvent)
     {
         var symbol = _symbolMapper.GetBrokerageSymbol(crossZeroOrderRequest.LeanOrder.Symbol);
         var tradeAction = ConvertDirection(crossZeroOrderRequest.LeanOrder.Direction, crossZeroOrderRequest.LeanOrder.SecurityType, crossZeroOrderRequest.OrderQuantityHolding);
 
         var response = _tradeStationApiClient.PlaceOrder(crossZeroOrderRequest.OrderType, crossZeroOrderRequest.LeanOrder.TimeInForce, Math.Abs(crossZeroOrderRequest.OrderQuantity), tradeAction, symbol, _accountType,
-            GetLimitPrice(crossZeroOrderRequest.LeanOrder), GetStopPrice(crossZeroOrderRequest.LeanOrder)).SynchronouslyAwaitTaskResult(); ;
+            GetLimitPrice(crossZeroOrderRequest.LeanOrder), GetStopPrice(crossZeroOrderRequest.LeanOrder)).SynchronouslyAwaitTaskResult();
 
         foreach (var error in response.Errors ?? Enumerable.Empty<TradeStationError>())
```

Comment:
Review `tradeStationApiClient.PlaceOrder` avoid duplication in implementation and caller location too, ideally it's a single generic method, or worst case scenario 1 method calling the other